homeassistant/components/sensor/sma.py

```diff
@@ -30,35 +30,28 @@
 CONF_SENSORS = 'sensors'
 CONF_UNIT = 'unit'
 
-GROUP_INSTALLER = 'installer'
-GROUP_USER = 'user'
-GROUPS = [GROUP_USER, GROUP_INSTALLER]
-
-SENSOR_OPTIONS = [
-    'current_consumption',
-    'current_power',
-    'total_consumption',
-    'total_yield',
-]
+GROUPS = ['user', 'installer']
 
 
 def _check_sensor_schema(conf):
     """Check sensors and attributes are valid."""
+    import pysma
```

Comment:
Requirements are not available when config validation is taking place.

Reply:
Thx, I asked in the chat, but had no reply there. Will update & move this to the setup routine